### DISCUSSION TITLE
After the newest Zotero 7 beta update, scrolling no longer working

### DISCUSSION THREAD

**Discussion by logicfollows** [windingwind/zotero-better-notes#874] [Q&A]
I have to disable Better Notes add-on in order to scroll to the bottom of the workspace note or even to some folders in my main library :( I am not sure if this issue is caused by the new layout of the newest Zotero 7 beta release, but I would really appreciate it of you could please help to fix this scrolling issue! I use Better Notes for everything and have been having to cut and copy my workspace note into smaller pieces and link it because of this scrolling issue.

**Comment by windingwind** [comment]
please use https://github.com/windingwind/zotero-better-notes/issues for bug reports

**Comment by logicfollows** [comment]
Thank you for the info, and I'll use that page in the future for bug reports. The issue is resolved when I open the note workspace in a new window, so it is no longer a problem.
